charts/harvester-csi-driver/harvester-csi-driver/0.1.400+up0.1.4/values.yaml

```diff
@@ -0,0 +1,45 @@
+# Default values for harvester-csi-driver.
+
+replicasCount: 3
+
+image:
+  harvester:
+    csiDriver:
+      repository: rancher/harvester-csi-driver
+      tag: ""
+  csi:
+    nodeDriverRegistrar:
+      repository: rancher/longhornio-csi-node-driver-registrar
+      tag: v2.3.0
+    resizer:
+      repository: rancher/longhornio-csi-resizer
+      tag: v1.2.0
+    provisioner:
+      repository: rancher/longhornio-csi-provisioner
+      tag: v2.1.2
+    attacher:
+      repository: rancher/longhornio-csi-attacher
+      tag: v3.2.1
```

Comment:
I believe we need mirrored images like in https://github.com/rancher/charts/blob/7bd8cb6123ec5ce9c4aec7f14917de6b3dd74370/charts/longhorn-1.2/longhorn/100.1.0%2Bup1.2.2/values.yaml#L28-L43...

Reply:
Both are available.
https://hub.docker.com/r/rancher/longhornio-csi-attacher/tags?page=1&name=v3.2.1
https://hub.docker.com/r/rancher/mirrored-longhornio-csi-attacher/tags?page=1&name=v3.2.1
Why do we want the mirrored prefix?

Reply:
We're just standardizing on using mirrored images when it is not owned by Rancher. And since Longhorn falls under its own organization, we generally stick with the mirrored images.

cc: @cbron if you think this is unnecessary?

Reply:
will continue conversation here: https://github.com/rancher/charts/pull/1595#discussion_r747763105

Reply:
so in conclusion we prefer to use the one with `mirrored-` prefix one only instead?

Reply:
Yes, if using longhorn images we should use the mirrored version.